fastembed/embedding.py

```diff
@@ -7,16 +9,20 @@
 from itertools import islice
 from multiprocessing import get_all_start_methods
 from pathlib import Path
-from typing import Any, Dict, Generator, Iterable, List, Optional, Tuple, Union
+from typing import Any, Callable, Dict, Generator, Iterable, List, Optional, Tuple, Union
 
 import numpy as np
 import onnxruntime as ort
 import requests
 from tokenizers import AddedToken, Tokenizer
 from tqdm import tqdm
+from huggingface_hub import snapshot_download
+from huggingface_hub.utils import RepositoryNotFoundError
 
 from fastembed.parallel_processor import ParallelWorkerPool, Worker
 
+logger = logging.getLogger(__name__)
```

Comment:
I'd prefer to have the [loguru](loguru.readthedocs.io/en/stable/api/logger.html) over the base logger because it also adds the function name for the log — useful context for debugging.